### DISCUSSION TITLE
File attachment

### DISCUSSION THREAD

**Discussion by x-o-r-r-o** [lockerpm/.github#9] [Ideas]
Hi,
I have been locker user since they launched, and I have requested via support to add File attachment feature in notes and other places with I think they have forgotten about this very major feature request so here I am submitting this request here to keep as a reminder.
Kindly expedite the addition of file attachment support in the locker, as my initial request was submitted quite some time ago.

Thank you

**Comment by Joey344532** [comment]
@x-o-r-r-o Thanks for reaching out!
We apologize for this late development. We have been through the UI revamping process for several platforms, so the other items needed to be put behind.
We are currently planning to implement the file attachment feature, which will soon appear. Please stay tuned for the updates!

**Reply by x-o-r-r-o** [reply]
@Joey344532 still waiting for this. Any chance of getting this feature?

**Reply by Joey344532** [reply]
Hi @x-o-r-r-o , happy to update that the file attachment feature has been released (for Premium users and above). Please try it out at https://passwords.locker.io/

**Comment by khanhnd157** [comment]
Attached files are now available
![image](https://github.com/user-attachments/assets/12a6e418-ad22-4939-a9ae-a663ddf5fda1)

**Reply by manofcult** [reply]
Thanks for this. However can we have an option to add multiple files (atleast 2) in the attachment.. Actually i am trying to add front and back of the business cards in the attachment. It would be really useful. Thanks

**Reply by Joey344532** [reply]
Hey @manofcult, you can add multiple attachments to an item by choosing the option **File attachment** of the item you want. Then upload or drag/drop the attachments to the form.
